### PR TITLE
Improved ClientStatisticsTest

### PR DESCRIPTION
**Pull request by Donnerbart**
* fixed a wrong cache name being used to retrieve the statistics
* added assert messages to ease debugging on test failures
* used more static imports

We had another NPE failure in the test, which I found hard to follow, since there were no assert messages in the test. While looking into the test, I found one stat assert, which used a wrong key to retrieve the cache hits.

### CODE REVIEW

File: hazelcast-client/src/test/java/com/hazelcast/client/statistics/ClientStatisticsTest.java [hunk 1]
```diff
@@ -72,91 +77,89 @@ public void cleanup() {
...
+                clientConnectionTime >= connectionTimeStat
+                        && clientConnectionTime - connectionTimeStat < STATS_PERIOD_SECONDS * 1000);
 
         String queueSize = stats.get("executionService.userExecutorQueueSize");
-        assertNotNull(queueSize);
+        assertNotNull(format("executionService.userExecutorQueueSize should not be null (%s)", stats), queueSize);
 
-        String mapHits = stats.get("nc." + testMapName + ".hits");
-        assertNull(mapHits);
-        String cacheHits = stats.get("nc." + testCacheName + ".hits");
```
Comment (Donnerbart) [inline hazelcast-client/src/test/java/com/hazelcast/client/statistics/ClientStatisticsTest.java]: Here the test used `"nc." + testCacheName + ".hits"`, which should be `"nc.hz/" + testCacheName + ".hits"` (like in all other lines).

**Comment by pveentjer** [comment]
Since this is just a test improvement and a simple one, one review is sufficient.
